adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -172,12 +172,16 @@
 
     static final String ACQUIRE_TOKEN_SILENT_SYNC_FORCE_REFRESH = "13";
 
+    static final String ACQUIRE_TOKEN_SILENT_SYNC_CLAIMS_CHALLENGE = "15";
```

Comment:
Has our API telemetry doc been updated with these values?

Reply:
Thanks @iambmelt . Updated the doc